content/robots.txt

```diff
@@ -1,4 +1,15 @@
 User-agent: *
-Allow: /
-Disallow: /wiki/
-Disallow: /slackintegrations
+Disallow: /
```

Comment:
Won't this exclude the home page from crawling?

Reply:
Yes. Is there anything on the home page that's not repeated on other pages (except gamification, which we don't want crawled anyway)?

Reply:
Well, the homepage itself is quite useful as a landing page (if we exclude it, search engines will give another page when searching for "Zeus WPI")

Reply:
True, I allowed the home page again in #75c8e6a.

Reply:
It would be nice if the root itself was also included (I think this is possible with the end-of-path terminator ($))

Reply:
Relevant the "$" is documented in the Google robots.txt spec: https://developers.google.com/search/reference/robots_txt

Reply:
I'm not quite visiting that page, but I'm pretty sure this is the right format. I do hope other crawlers understand this, too.

Reply:
Maybe also allow sponsorende leden, so websites of sponsorende leden get better pagerank index (from ugent subdomain)